File: CC/Codigos_Fonte_Relatorio_1/micro08/micro08.pas
program micro08;

var
    numero : integer;


begin
    numero := 1;

    while (numero <> 0) do
    begin
        write('Digite um número: ');
        read(numero);   

        if(numero > 10) then
            writeln('O número ',numero,' é maior que dez.')
        else
            writeln('O número ',numero, ' é menor que dez.')
    end;
end.

File: CC/Codigos_Fonte_Relatorio_1/micro08/micro08.java

public class micro08{

    public static void main(String[] args){
        int numero;

        numero = 1;

        while(numero != 0){
            System.out.println("Digite um número: ");
            numero = Integer.parseInt(System.console().readLine());

            if(numero > 10 ) System.out.println("O numero "+numero+" é maior que dez.");
            else System.out.println("O numero "+numero+" é menor que dez.");
         
        }
    }
}
